lib/brewdler/utils.rb

```diff
@@ -8,11 +8,28 @@ def self.system cmd, *args
     $?.success?
   end
 
-  def self.brew_installed?
-    system("brew --version &>/dev/null")
+  def brew_installed?
+    Kernel.system("brew --version &>/dev/null")
   end
 
-  def self.cask_installed?
-    system("brew cask --version &>/dev/null")
+  def cask_installed?
+    Kernel.system("brew cask --version &>/dev/null")
+  end
+
+  def brewfile
+    File.read(Dir['{*,.*}{B,b}rewfile'].first.to_s)
+  rescue Errno::ENOENT
+    raise "No Brewfile found"
+  end
+
+  def getc  # NOTE only tested on OS X
+    Kernel.system "/bin/stty raw -echo"
+    if STDIN.respond_to?(:getbyte)
+      STDIN.getbyte
+    else
+      STDIN.getc
+    end
+  ensure
+    Kernel.system "/bin/stty -raw echo"
   end
```

Comment:
> Could maybe make it not require --force for removing packages that were installed from bottles because it's trivial to reinstall them?

That's a whole different feature which is complicated and not very needed. :santa:

> I fear people will want them in more places.

There will be some place interactive thing has more benefit. In fact, this code is copied from homebrew install script. And quite frankly, this repo is not homebrew core.

I kinda want to hear more people's opinions on this. cc @andrew If indeed majority people hate interactive, I would suggest to implement this as `cleanup` and `cleanup --dry-run`. Then maybe I put interactive script into my own tap.